### PR TITLE
Star rating

### PR DESCRIPTION
**Pull request by Alexander-Schulte**
hello again! :) 
In this PR I created the courtRating! You can give 1 - 5 stars for a court at the addCourt page. While the user clicks on a star, the color of the stars should change. After the Court was added, the given rating should be seen at the CourtList page.

thank u for your review! :)

### CODE REVIEW

File: src/components/CourtCardsList.js [hunk 1]
```diff
@@ -40,6 +41,20 @@ function CourtList() {
...
+              {[...Array(5)].map((star, i) => {
+                const ratingValue = i + 1;
+                return (
+                  <FaStar
+                    className="addStarIcon"
+                    size={20}
+                    color={ratingValue <= court.rating ? "#ffc107" : "#e4e5e9"}
+                  />
+                );
+              })}
```
Comment (FriederikeM) [inline src/components/CourtCardsList.js:55]: What is happeniiiing? Back arrows? Looks very professional. You'#ll have to explain it to me :D

**Review by FriederikeM — APPROVED** [review]
Looks very good, Alex. I can't really comment, because this is too smart for me 🤓

File: src/components/StarRating.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import { FaStar } from "react-icons/fa";
+import "./CourtCards.css";
+export function StarRating({ onRatingChange, value }) {
+  return (
+    <div className="addGroupRating">
+      {[...Array(5)].map((star, i) => {
```
Comment (yvlange) [inline src/components/StarRating.js:6]: You're definitely turning into a map master ;)

File: src/components/StarRating.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+              className="radio"
+              type="radio"
+              name="starRating"
+              value={ratingValue}
+              onClick={handleOnCLick}
+            />
+            <FaStar
+              className="addStarIcon"
+              color={ratingValue <= value ? "#ffc107" : "#e4e5e9"}
+              size={20}
```
Comment (yvlange) [inline src/components/StarRating.js:25]: Great to see that you can put the styling of icons in here. Quite enlightning :)

**Review by yvlange — APPROVED** [review]
Wow great job Alex! You definitely did some methods I haven't seen yet and it works out pretty well 👍  The code looks quite clean to me too, so I have nothing to optimize :)

File: src/components/CourtCardsList.js (hunk 1 above)
Comment (Alexander-Schulte) [reply src/components/CourtCardsList.js:55]: Thank u very much! I'll show you! :)

**Pull request merged**